recipes/goftests/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "goftests" %}
+{% set version = "0.2.3" %}
+{% set sha256 = "786e6877ac327576a07a50da47a28df28340cac8028cfd992660234ab34f5910" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - numpy
+    - scipy
+    - scikit-learn
+
+test:
+  imports:
+    - goftests
+
+about:
+  home: https://github.com/posterior/goftests
+  license: BSD-3-Clause
+  license_family: BSD
```

Comment:
Thanks. Please update this recipe to include `license_file: LICENSE.txt` on the next release that has that change.